selfdrive/car/honda/carcontroller.py

```diff
@@ -76,6 +77,13 @@ def brake_pump_hysteresis(apply_brake, apply_brake_last, last_pump_ts, ts):
   return pump_on, last_pump_ts
 
 
+def round_hud_set_speed(v_cruise_kph, is_metric):
+  if not is_metric:
+    v_cruise_kph = v_cruise_kph / 1.00584
```

Comment:
```suggestion
    v_cruise_kph = v_cruise_kph / (CV.MPH_TO_KPH / IMPERIAL_INCREMENT)
```

might be this

Reply:
Nice! This PR + the suggested change for line 82 mapped properly from 5 to 80 mph for me on 2017 Civic Hatch today. Imperial unit is 1 on this car. 

One problem though: when disengaged, the sent bus signal was 253 instead of 255.

Reply:
I fixed this like 4 years ago on my fork.  I couldn't figure out how they are converting their values so I made a spreadsheet with all possible values and just did a line of best fit to get a formula lol.  Ended up with: int(round((float(self.v_cruise_kph) * 0.6233 + 0.0995)))

This is the table I got from Aragon's Nidec Honda Civic:
KM/H -> MPH
--------
40 = 25
42 = 26
43 = 27
45 = 28
46 = 29
48 = 30
50 = 31
51 = 32
53 = 33
54 = 34
56 = 35
58 = 36
59 = 37
61 = 38
62 = 39
64 = 40
66 = 41
67 = 42
69 = 43
70 = 44
72 = 45
74 = 46
75 = 47
77 = 48
78 = 49
80 = 50
82 = 51
83 = 52
85 = 53
86 = 54
88 = 55
90 = 56
91 = 57
93 = 58
94 = 59
96 = 60
98 = 61
99 = 62
101 = 63
103 = 64
104 = 65
106 = 66
107 = 67
109 = 68
111 = 69
112 = 70
114 = 71
115 = 72
117 = 73
119 = 74
120 = 75
122 = 76
123 = 77
125 = 78
127 = 79
128 = 80
130 = 81
131 = 82
133 = 83
135 = 84
136 = 85
138 = 86
139 = 87
141 = 88
143 = 89
144 = 90